packages/builder/src/components/backend/modals/DeleteDataConfirmationModal.svelte

```diff
@@ -85,7 +86,7 @@
 
   async function deleteView(view: ViewV2 | View) {
     try {
-      if ("version" in view && view.version === 2) {
+      if ("version" in view && helpers.views.isCalculationView(view)) {
```

Comment:
Is this a mistake? I'm not sure calculation view is the right check here.

Reply:
Ah woops - auto-complete has led me astray!